sections/product-recommendations.liquid

```diff
@@ -22,24 +22,26 @@
   }
 {%- endstyle -%}
 
-<product-recommendations class="product-recommendations page-width section-{{ section.id }}-padding" data-url="{{ routes.product_recommendations_url }}?section_id={{ section.id }}&product_id={{ product.id }}&limit=4">
-  {% if recommendations.performed and recommendations.products_count > 0 %}
-    <h2 class="product-recommendations__heading">{{ section.settings.heading | escape }}</h2>
-    <ul class="grid grid--2-col product-grid{% if recommendations.products_count > 3 %} grid--4-col-desktop grid--quarter-max{% else %} grid--{{ recommendations.products_count }}-col-tablet{% endif %}" role="list">
-      {% for recommendation in recommendations.products %}
-        <li class="grid__item">
-          {% render 'card-product',
-            card_product: recommendation,
-            media_aspect_ratio: section.settings.image_ratio,
-            show_secondary_image: section.settings.show_secondary_image,
-            show_vendor: section.settings.show_vendor,
-            show_rating: section.settings.show_rating
-          %}
-        </li>
-      {% endfor %}
-    </ul>
-  {% endif %}
-</product-recommendations>
+<div class="color-{{ section.settings.color_scheme }} gradient">
```

Comment:
This section may remain visible on the page even if there are no product recommendations returned by the server. This is actually due to the section padding settings, but is now more noticeable when a different color scheme is selected.

![Screen Shot 2022-01-10 at 11 59 12 2 AM](https://user-images.githubusercontent.com/3710548/148807071-2ac3b6ae-db4c-4b1e-b793-c64462c941b1.png)